README.md

```diff
@@ -47,16 +47,18 @@ Possible options for the reporter:
 
  - `:reporter_name` - unique name for the reporter. The purpose is to distinguish between different reporters running in the system.
     One can run separate independent InfluxDB reporters, with different configurations and goals.
- -  `:protocol` - :udp or :http. Which protocol to use for connecting to InfluxDB. Default option is :udp.
- -  `:host` - host, where InfluxDB is running.
- -  `:port` - port, where InfluxDB is running.
- -  `:db` - name of InfluxDB's  instance.
- -  `:username` - username of InfluxDB's user that has writes privileges.
- -   `:password` - password for the user.
+ - `:protocol` - :udp or :http. Which protocol to use for connecting to InfluxDB. Default option is :udp.
+ - `:host` - host, where InfluxDB is running.
+ - `:port` - port, where InfluxDB is running.
+ - `:db` - name of InfluxDB's  instance.
+ - `:username` - username of InfluxDB's user that has writes privileges.
+ - `:password` - password for the user.
  - `:events` - list of `Telemetry` events' names that we want to send to InfluxDB.
     Each event should be specified by the map with the field `name`, e.g. `%{name: [:sample, :event, :name]}`.
     Event names should be compatible with `Telemetry` events' format.
- - `:tags` - list of global tags, that will be attached to each reported event. The format is a map,
+    It is also possible to specify an optional list of metadata keys that will be included in the event body and sent to InfluxDB as tags.
+    The list of metadata keys should be specified in the event data with the field `metadata_keys`, e.g. `%{name: [:sample, :event, :name], metadata_keys: [:sample_meta, sample_meta2]}`
```

Comment:
just thinking if `metadata_tag_keys` is more explicit name

Reply:
dont have a strong opinion though

Reply:
You are right, I renamed the variable as you suggested.